Fix/pdf generation

Naeemedmo opened the pull request:
Remove space that causes errors in pdf generation. (See https://tex.stackexchange.com/questions/233249/unknown-graphics-extension-png-pdf for a description of the problem and the solution)

Remove hline between technical skills and adapt the vertical spacing in this section so that it fits the rest.

jonathf commented:
Bugfix and stretch looks good.

The vertical bars was just added in #52 by suggested from @kememil. I see why you want to remove them, but see the argument for keeping them as well.

I think after a little consideration the best course of action is to return to how it was before. We are very close as the icons are off by default.

Do you mind replacing the tech-skill seperator from `\nobreak\divbar{}` to just `, `?

Naeemedmo commented:
In my last commit, I removed the horizontal line. I think if we start adding horizontal sublines within the technical section, then we need them in other sections too, to keep the style consistent, which would make the whole document a bit crowded in my option. 

I replaced the divbars with a comma as you suggested. I agree that it does look better, as it also matches the interests and hobbies section. But I kind of liked the divbars. Another option would be to add them to the interest section too. What do you think? 

I am looking forward to working with you from September :-)
